docs-kits/kits/Traceability Kit/Software Development View/page_software-development-view.md

```diff
@@ -386,55 +442,284 @@ If no access control shall be applied, externalSubjectId must be omitted (no acc
-- `endpoints`: For access of submodels via EDC and AAS a new interface type EDC-AAS must be defined.
-  - `interface`: The value must be "EDC"
-  - `endpointAddress`: The endpoint address must have the following format: `http://provider.controlplane:port/<EDC Asset ID>/submodel?content=value&extent=WithBLOBValue`
-    - `provider.controlplane:port`: server and port of the EDC that is providing the submodel
-    - `EDC Asset ID`: The EDC asset id under which the submodel was registered in the EDC. It must have the following format `<AAS ID>-<Submodel ID>`
-      - `AAS ID`: The id of the digital twin (identification property in the AAS descriptor)
-      - `Submodel`: The id of the submodel  (identification property in the submodel descriptor)
-    - `/submodel`: This method from the AAS Shell Interface that is invoked via the EDC.
-    - `content=value&extent=WithBLOBValue`: This are currently required parameters when requesting payload via the AAS standard from an EDC
-  - `endpointProtocol`: The value must be `IDS/ECLIPSE DATASPACE CONNECTOR`
-  - `endpointProtocolVersion`: The value must be `0.0.1-SNAPSHOT`
-
-> :raised_hand: **AAS Submodel Descriptor Endpoints**
-The endpoint (endpointAddress) in the submodel descriptor cannot be used directly to contact the EDC and access the data. The endpoint has to be re-written as it combines EDC and AAS information. The AAS API Wrapper does that automatically when using it, i.e., it fetches the endpoint from the AAS Registry and re-writes it in a way to a Catena-X EDC with an AAS server in the background can process the query.
+General conventions for submodel descriptors can be found in the standard [CX-0002 Digital Twins in Catena-X](https://catena-x.net/de/standard-library). This section is based on it and it is recommended to read it first. In this KIT, we extend this standard with some additional conventions.
+
+Submodel descriptors MUST be compliant to the following additional conventions:
+
+- `id`: The submodel ID must be a UUIDv4 in URN format: "urn:uuid:&lt;UUIDv4&gt;";
+- `idShort`: the name of the aspect model in camel case, e.g. for aspect SerialPart: "serialPart".
+
+The actual access information for the EDC is part of the endpoint attribute in the submodel descriptor.
+
+```json
+{
+    "interface": "SUBMODEL-3.0",
+    "protocolInformation": {
+        "href": "https://edc.data.plane/{path}/submodel",
+        "endpointProtocol": "HTTP",
+        "endpointProtocolVersion": ["1.1"],
+        "subprotocol": "DSP",
+        "subprotocolBody": "id=123;dspEndpoint=http://edc.control.plane/",
+        "subprotocolBodyEncoding": "plain",
+        "securityAttributes": [ 
+          { "type": "NONE", "key": "NONE", "value": "NONE" }
+        ]
+    }
+}
+```
+
+The following conventions apply for the endpoint:
+
+- `interface`, `endpointProtocol`, `endpointProtocolVersion`, `subprotocol`, `subprotocolBodyEncoding`, and `securityAttributes` are set as defined in the CX-0002 standard.
```

Comment:
Is the `CX-002` Standard already publicly available? If so, I think a link would help the reader. 
Potentially all references to the CX standards should link to the source.

Reply:
I just added links to the location of the standards.